lib/Interpreter/CIFactory.cpp

```diff
@@ -42,9 +43,19 @@
 
 #include <memory>
 
+#ifndef _MSC_VER
+#include "cling-compiledata.h"
+# include <unistd.h>
+# define getcwd_func getcwd
```

Comment:
Why not just use LLVM's . Wouldn't it be nice if this could be integrated into their codebase one day?